builder/builder-next/exporter/writer.go

```diff
@@ -20,13 +21,25 @@ import (
 // )
 
 func emptyImageConfig() ([]byte, error) {
-	img := ocispec.Image{
-		Architecture: runtime.GOARCH,
-		OS:           runtime.GOOS,
+	pl := platforms.Normalize(platforms.DefaultSpec())
+
+	type image struct {
+		ocispec.Image
+
+		// Variant defines platform variant. To be added to OCI.
+		Variant string `json:"variant,omitempty"`
```

Comment:
Looks like the vendoring is updated in this PR to a version that has the `Variant` field; so we probably don't need this custom struct? https://github.com/moby/moby/pull/43239/commits/7eb4ef66775075b9c644155ccab8ae5e08d3e51c#diff-cbddc5af26a81e622cffefb37996ea04e1b66aa3a811d8b9e9b291ffb8195d19R92-R94

Reply:
Done